function-data-migration/src/main/java/com/future/function/data/migration/change/log/DataMigration_020.java

```diff
@@ -0,0 +1,77 @@
+package com.future.function.data.migration.change.log;
+
+import com.future.function.common.enumeration.core.Role;
+import com.future.function.model.entity.feature.core.Menu;
+import com.future.function.model.util.constant.FieldName;
+import com.github.mongobee.changeset.ChangeLog;
```

Comment:
I think we should use `mongock`, not `mongobee` anymore. cc @DavidWilliamK

Reply:
Please pull from `release/2020-01-13` to use `mongock`